packages/runtime/src/types.ts

```diff
@@ -185,8 +185,6 @@ export interface GatewayHiveCDNOptions {
 export interface GatewayHiveReportingOptions
   extends Omit<YamlConfig.HivePlugin, 'experimental__persistedDocuments'> {
   type: 'hive';
-  /** GraphQL Hive registry access token. */
-  token: string;
```

Comment:
There is no CLI or CLI arguments in the runtime. The `token` is still required in the runtime.

Reply:
Then we need to make it optional for CLI only?

Reply:
How about now? @enisdenjo